v21.1/cockroach-debug-ballast.md

```diff
@@ -3,13 +3,14 @@ title: cockroach debug ballast
 summary: Create a large, unused file in a node's storage directory that you can delete if the node runs out of disk space.
 toc: true
 redirect_from: debug-ballast.html
-key: debug-ballast.html
 ---
 
-The `cockroach debug ballast` [command](cockroach-commands.html) creates a large, unused file that you can place in a node's storage directory. In the case that a node runs out of disk space and shuts down, you can delete the ballast file to free up enough space to be able to restart the node.
+The `cockroach debug ballast` [command](cockroach-commands.html) creates a large, unused file that you can place in a node's storage directory. In the case that a node runs out of disk space and shuts down, you can delete the ballast file to free up enough space to be able to restart the node.  
 
+- Do not run `cockroach debug ballast` with a unix `root` user. Doing so brings the risk of mistakenly affecting system directories or files.
+- <span class="version-tag">Changed in v21.1:</span> `cockroach debug ballast` now refuses to overwrite the target ballast file if it already exists. This change is intended to prevent mistaken uses of the `ballast` command. Consider adding an `rm` command to scripts that integrate `cockroach debug ballast`, or provide a new file name every time and then remove the old file.
+- Do not name the target file similar to a block device in `/dev`. Doing so brings the risk of mistyping a `/dev` prefix into the command and thereby corrupting a filesystem.  
```

Comment:
this last point is not necessary any more, since the command will now avoid overwriting/writing to something that already exist.

This recommendation should be mentioned in the docs for the versions where the change was not backported, though.

Reply:
Done. PTAL.